dialog/color.go

```diff
@@ -166,16 +167,30 @@ func newColorButtonBox(colors []color.Color, icon fyne.Resource, callback func(c
 	return container.NewGridWithColumns(8, objects...)
 }
 
-func newCheckeredBackground() *canvas.Raster {
-	return canvas.NewRasterWithPixels(func(x, y, _, _ int) color.Color {
-		const boxSize = 8
+const (
```

Comment:
Can you move the const to the top of the file? We try to keep things in order as: https://github.com/fyne-io/fyne/wiki/Contributing